float32 pid_pos_out_1
int16 moto_chassis_info_1
int16 set_spd_info_1

float32 pid_pos_out_2
int16 moto_chassis_info_2
int16 set_spd_info_2


float32 pid_pos_out_3
int16 moto_chassis_info_3
int16 set_spd_info_3

float32 pid_pos_out_4
int16 moto_chassis_info_4
int16 set_spd_info_4

float32 pid_pos_out_5
int16 moto_chassis_info_5
int16 set_spd_info_5

float32 pid_pos_out_6
int16 moto_chassis_info_6
int16 set_spd_info_6

